examples/ai-core/src/generate-transcript/openai.ts

```diff
@@ -0,0 +1,21 @@
+import { openai } from '@ai-sdk/openai';
+import { experimental_transcribe as transcribe } from 'ai';
+import 'dotenv/config';
+import { readFile } from 'fs/promises';
+
+async function main() {
+  const result = await transcribe({
+    model: openai.transcription('whisper-1'),
+    audio: await readFile('audio.mp3'),
```

Comment:
we have existing data files so examples just work; you could use e.g. `examples/ai-core/data/galileo.mp3`